pkg/webui/console/containers/application-overview-header/index.js

```diff
@@ -204,6 +240,26 @@ const ApplicationOverviewHeader = () => {
             dropdownPosition="below left"
           />
         </div>
+        <RequireRequest
+          requestAction={[getPubsubsList(application_id), getWebhooksList(application_id)]}
+        >
+          <DeleteEntityHeaderModal
+            entity="application"
```

Comment:
Please always use the `/pkg/webui/console/constants/entities.js` module to reference entity type names. This ensures consistency within possible comparisons.

In doing so, you could also move all the selectors, actions, rights checks, etc into the modal component directly and choose them based on a switch of the entity type. I don't insist on this but suggest you to consider this as it would clean up the overview header files significantly, which now have a very heavy focus on the deletion logic.

```jsx
<DeleteEntityHeaderModal
  entity={APPLICATION}
  entityId={application_id}
  entityName={name}
  setVisible={setDeleteApplicationModalVisible}
  visible={deleteApplicationModalVisible}
/>
```